elasticdl/python/common/model_handler.py

```diff
@@ -2,67 +2,83 @@
 
 import tensorflow as tf
 
+from elasticdl.proto import elasticdl_pb2
+from elasticdl.python.common import model_utils
 from elasticdl.python.common.constants import DistributionStrategy
 from elasticdl.python.common.log_utils import default_logger as logger
+from elasticdl.python.common.tensor import Tensor
 from elasticdl.python.elasticdl.layers.embedding import Embedding
 
 
 class ModelHandler(metaclass=abc.ABCMeta):
-    """
-    Generate the model to train in ElasticDL for different distributed
+    """Generate the model to train in ElasticDL for different distributed
     strategies and export trained model in ElasticDL to SavedModel.
     """
 
     @abc.abstractmethod
     def get_model_to_train(self, model):
+        """Generate a model to train in ElasticDL.
+
+        Args:
+            model: A keras model instance.
+
+        Returns:
+            A keras model instance.
         """
-        Generate a model to train in ElasticDL.
-        """
-        pass
 
     @abc.abstractmethod
-    def get_model_to_export(self, model, trained_params, dataset):
-        """
-        Get the model which can be exported a SavedModel
+    def get_model_to_export(self, model, dataset):
+        """Get the model which can be exported a SavedModel
         by tf.saved_model.save.
+
+        Args:
+            model: A keras model instance.
```

Comment:
A keras model instance trained by ElasticDL.

Reply:
The `model` in args is the native Keras model and the returned model is the trained model.